_targets.R

```diff
@@ -6,10 +6,64 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+# Vector of stations for which to download data
+site_nums <- c("01427207", "01432160", "01435000", "01436690", "01466500")
+# Plot arguments
+p_width = 12
+p_height = 7
+p_units = 'in'
+# dummy variable to force re-downloading data from web
+dummy_date = '2021-08-30'
```

Comment:
Not a requirement in this course, but definitely something we do in our big data download pipelines!